packages/slo-stat-bot/src/slo-lint.ts

```diff
@@ -145,18 +147,57 @@ async function getFileShaContents(
  */
 export const lint = async function lint(
   schema: JSON,
-  sloData: JSON
+  sloData: SLORules[]
 ): Promise<ValidationResults> {
   const ajv = new Ajv();
   const validate = ajv.compile(schema);
-  const isValid = await validate(sloData);
+  let isValid = await validate(sloData);
+
+  if (validate.errors) {
+    return {
+      isValid: isValid,
+      errors: validate.errors,
+    } as ValidationResults;
+  }
+
+  //Check for contradicting SLOs
+  let errors = null;
```

Comment:
You typically don't need to explicitly assign to `null` like this!  Using `let errors: Error[];` will default to `undefined`, which in most cases here is perfectly ok